requirements.txt

```diff
@@ -2,6 +2,7 @@ geolinks==0.2.0
 lxml==3.6.2
 OWSLib==0.13.0
 pyproj==1.9.5.1
+setuptools
```

Comment:
It seems it became part of standardlib [only after python 2.7.9](https://packaging.python.org/installing/#install-pip-setuptools-and-wheel), but yeah, since we've dropped py26 earlier this may not be necessary. I'll remove.